website/project/model.py

```diff
@@ -609,6 +611,15 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin):
     description = fields.StringField()
     category = fields.StringField(validate=validate_category, index=True)
 
+    # Representation of a nodes's license
+    # {
+    #  'id':  <id>,
+    #  'name': <name>,
+    #  'text': <license text>,
+    #  'properties' (optional): ['Year' (optional), 'Copyright Holders' (optional)]
```

Comment:
Oh, I think I see: `properties` are the the human-readable "labels" for extra information about the license.